Simplenote/metadata/PlayStoreStrings.pot

```diff
@@ -11,17 +11,19 @@ msgstr ""
 "Project-Id-Version: Release Notes & Play Store Descriptions\n"
 
 #. translators: Release notes for this version to be displayed in the Play Store. Limit to 500 characters including spaces and commas!
-msgctxt "release_note_029"
+msgctxt "release_note_0210"
 msgid ""
-"2.9:\n"
-"* Fixed a bug with duplicate tags and special characters in tag names\n"
+"2.10:\n"
+"* Added improvements for devices with hardware keyboards\n"
+"* Added support for screen readers when adding a new note\n"
+"* Fixed an issue with font size resizing when swiping\n"
```

Comment:
```suggestion
```

I'm not sure if we need to remove this item from the release notes here too if we decide to remove it from the `RELEASE_NOTES.txt` file as well or if it's automatically removed with our release scripts.